BibBuddy/app/src/main/java/de/bibbuddy/BookNotesViewModel.java

```diff
@@ -27,11 +27,11 @@ public BookNotesViewModel(Context context) {
         noteList = new ArrayList<>();
         for (Note item : list) {
             if (item.getType() == 1) { //text
-                noteList.add(new NoteItem(item.getName(), item.getId()));
+               // noteList.add(new NoteItem(item.getName(), item.getId()));
```

Comment:
Why is this commented out?

Reply:
This was causing bugs and entire class needed to be adjusted to noteModel-usage. This should be done now.